go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/kyverno/kyverno
 
 go 1.20
 
+replace github.com/google/go-containerregistry/0_14 => github.com/google/go-containerregistry v0.14.1-0.20230425172351-b7c6e9dc3944
```

Comment:
👍

You may want to combine this with other replaced packages, I'm seeing a conflict in this file.